frontend/src/store.ts

```
@@ -139,33 +177,33 @@ export function useStore() {
 
       const currentRequestId = this.requestID();
 
       for (const id in res.Requests) {
         if (id !== currentRequestId) {
-          requests[id] = res.Requests[id];
+          this.requests[id] = res.Requests[id];
         }
       }
-      for (const id in requests) {
+      for (const id in this.requests) {
         if (!res.Requests.hasOwnProperty(id)) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)